[DE] Update rules in the ASC (#7431)
ppy/osu-wiki: wiki/Article_styling_criteria/Writing/de.md

### Geviertstrich

Geviertstriche (`—`) müssen mit einem Leerzeichen davor und dahinter verwendet werden. In [Blockzitaten](#blockzitate) haben Geviertstriche nur ein Leerzeichen auf ihrer rechten Seite. Bindestriche (`-`) dürfen nicht an Stelle von Geviertstrichen verwendet werden.

### Zitate

Verweis auf [Wikipedias Artikel zum Format für Zitate](https://de.wikipedia.org/wiki/Wikipedia:Zitate#Beispiele).

> Füge abschließende Satzzeichen nur dann in die Anführungszeichen ein, wenn sie im Originaltext vorhanden waren und setze sie andernfalls nach dem schließenden Anführungszeichen. In den meisten Fällen bedeutet dies, dass Punkt und Komma genauso behandelt werden wie Fragezeichen: Sie stehen innerhalb der Anführungszeichen, wenn sie sich nur auf das zitierte Material beziehen, und außerhalb, wenn sie sich auf den gesamten Satz beziehen.

Reviewer: This block is actually marked as a quote in the original (english) article, serving as the example that is referred to below. 
![image](https://user-images.githubusercontent.com/22643729/171900219-69114b22-891e-4da1-b6dc-59adda509cbd.png)
It's unfortunate that the german style guide linked here here doesn't feature an equivalent paragraph. Maybe we should add some other quote to serve as an example.

Reviewer 2: you can mention in parentheses that the quoted material is translated from english, and also link to the english article

Reviewer: ```suggestion
Übersetzt aus dem englischen [Wikipedia Manual of Style](https://en.wikipedia.org
/wiki/Wikipedia:Manual_of_Style#Punctuation_inside_or_outside):

> Füge abschließende Satzzeichen nur dann in die Anführungszeichen ein, wenn sie im Originaltext vorhanden waren und setze sie andernfalls nach dem schließenden Anführungszeichen. In den meisten Fällen bedeutet dies, dass Punkt und Komma genauso behandelt werden wie Fragezeichen: Sie stehen innerhalb der Anführungszeichen, wenn sie sich nur auf das zitierte Material beziehen, und außerhalb, wenn sie sich auf den gesamten Satz beziehen.
```

If you want to follow the rule below about how to present sources it would have to be like this right?

Author: I think it's fine like this. Though, there is a line break in the link. Can you please remove it?

Another point is if we need to have links to both the German and English variant of the Wikipedia citation article 🤔

Author: I will apply the suggestion from @FamousPig as that is a bit shorter and additionally remove the reference to the German variant. Having both versions in the ASC may be confusing regarding what to use due to the differences between the two variants. If we ever need to specify more rules for citations (and thus change this section), we can do so later.

> Verweis auf [Wikipedias Artikel zum Format für Zitate](https://de.wikipedia.org/wiki/Wikipedia:Zitate#Beispiele).
> 
> Füge abschließende Satzzeichen nur dann in die Anführungszeichen ein, wenn sie im Originaltext vorhanden waren und setze sie andernfalls nach dem schließenden Anführungszeichen. In den meisten Fällen bedeutet dies, dass Punkt und Komma genauso behandelt werden wie Fragezeichen: Sie stehen innerhalb der Anführungszeichen, wenn sie sich nur auf das zitierte Material beziehen, und außerhalb, wenn sie sich auf den gesamten Satz beziehen.

Reviewer 2: like this?

```suggestion
Verweis auf [Wikipedias Artikel zum Format für Zitate](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style#Punctuation_inside_or_outside) (aus dem englischen übersetzt).

> Füge abschließende Satzzeichen nur dann in die Anführungszeichen ein, wenn sie im Originaltext vorhanden waren und setze sie andernfalls nach dem schließenden Anführungszeichen. In den meisten Fällen bedeutet dies, dass Punkt und Komma genauso behandelt werden wie Fragezeichen: Sie stehen innerhalb der Anführungszeichen, wenn sie sich nur auf das zitierte Material beziehen, und außerhalb, wenn sie sich auf den gesamten Satz beziehen.
```

Im weiteren Sinne sollten [Betonungen](/wiki/Article_styling_criteria/Formatting#betonung), [Links](/wiki/Article_styling_criteria/Formatting#links) und Klammern denselben Regeln bezogen auf die Zeichensetzung folgen (also, ob sie einen ganzen Satz oder nur einen Teil davon miteinbeziehen).

### Blockzitate

> Wenn Text von jemandem zitiert wird, sollte die zitierte Person nach dem [Blockzitat](/wiki/Article_styling_criteria/Formatting#blockzitate) mit einem Geviertstrich (`—`) und einem Leerzeichen angegeben werden. Sofern das Originalzitat nicht aus dem Englischen stammt, spezifiziere die Sprache in Klammern.

Reviewer: > Wait a moment, not sure if you meant this but the sentence above should mention non-German instead of non-English 
> texts. Aus dem Deutschen übersetzt doesn't make sense at all!

That is all I meant

> Wenn Text von jemandem zitiert wird, sollte die zitierte Person nach dem [Blockzitat](/wiki/Article_styling_criteria/Formatting#blockzitate) mit einem Geviertstrich (`—`) und einem Leerzeichen angegeben werden. Sofern das Originalzitat nicht aus dem Englischen stammt, spezifiziere die Sprache in Klammern.

Author: ```suggestion
Wenn Text von jemandem zitiert wird, sollte die zitierte Person nach dem [Blockzitat](/wiki/Article_styling_criteria/Formatting#blockzitate) mit einem Geviertstrich (`—`) und einem Leerzeichen angegeben werden. Sofern das Originalzitat nicht aus dem Deutschen stammt, spezifiziere die Sprache in Klammern.
```

```markdown
> bitte habt Spaß am Spiel

— rrtyui (aus dem {Sprache} übersetzt)
```

> Im Allgemeinen sollte die Quelle des Zitats darüber spezifiziert werden. Siehe die [obere Sektion](#zitate) für ein Beispiel. Beim Zitieren eines schriftlichen Artikels oder anderweitig, und nicht unbedingt eine spezifische Person, muss der Autor nicht unter dem Blockzitat angegeben werden.

Author: `darüber` sounds weird/ambigious in this context. I'm sure this is refering to the generated superscripts. Is there a different phrase that expresses this in a better way?

> Im Allgemeinen sollte die Quelle des Zitats über diesem spezifiziert werden. Siehe die [obere Sektion](#zitate) für ein Beispiel. Beim Zitieren eines schriftlichen Artikels oder anderweitig, und nicht unbedingt eine spezifische Person, muss der Autor nicht unter dem Blockzitat angegeben werden.

Author: ```suggestion
Im Allgemeinen sollte die Quelle des Zitats über diesem spezifiziert werden. Siehe die [obere Sektion](#zitate) für ein Beispiel. Beim Zitieren eines schriftlichen Artikels oder anderweitig, und nicht unbedingt einer spezifischen Person, muss der Autor nicht unter dem Blockzitat angegeben werden.
```
I would say it's `Zitieren einer spezifischen Person`.
